lib/sprockets/commoner/processor.rb

```diff
@@ -44,20 +47,14 @@ def self.call(input)
       end
 
       attr_reader :include, :exclude, :babel_exclude
-      def initialize(root, include: [root], exclude: [], babel_exclude: [/node_modules/])
+      def initialize(root, include: [root], exclude: ['vendor/bundle'], babel_exclude: [/node_modules/])
```

Comment:
Sorry in case it wasn't more clear: CircleCI installs the gems under `vendor/bundle` which means commoner processes them, while normally it would ignore them as they'd be outside the rails application root